src/plugins/intel_npu/src/compiler/src/driver_compiler_adapter.cpp

```diff
@@ -115,8 +115,7 @@ void LevelZeroCompilerAdapter::release(std::shared_ptr<const NetworkDescription>
     apiAdapter->release(std::move(networkDescription));
 }
 
-CompiledNetwork LevelZeroCompilerAdapter::getCompiledNetwork(
-    std::shared_ptr<const NetworkDescription> networkDescription) {
+CompiledNetwork LevelZeroCompilerAdapter::getCompiledNetwork(const NetworkDescription& networkDescription) {
     _logger.info("getCompiledNetwork - using adapter to perform getCompiledNetwork(networkDescription)");
     return apiAdapter->getCompiledNetwork(std::move(networkDescription));
```

Comment:
Here something strange happening, you move from reference to const - it's a copy anyway, so move probably should removed to keep code clear - not a bug, though